workloads/jenkins/Jenkinsfile

```diff
@@ -83,6 +83,11 @@ pipeline {
             timeout 180 bash -x ./workloads/jenkins/scripts/test-dashboard.sh
           '''
 
+          echo "Start new virtual environment pre-Slurm checks"
```

Comment:
Do we need to explicitly tear down the VMs before we do this?

Reply:
Tearing them down is part of this script.